libs/clip/clip/Clipboard.hx

```diff
@@ -0,0 +1,20 @@
+package clip;
```

Comment:
A whole top-level package just for clipboard? :/

Reply:
Yeah, thought that seems like a bit much. I kinda just followed the sdl -libs scheme and added it's own. Also, in the future you probably want more clipboard functionality (images etc), so there probably won't be only text related functionality there...

What would you suggest for the package name?